controlpanel/interfaces/web/templates/base.html

```diff
@@ -10,8 +10,8 @@
 
 {% block header %}
     {% gds_header homepageUrl="/" serviceName="Data platform Control panel" serviceUrl="/" containerClasses="govuk-width-container" %}
-        {% gds_header_nav_item href="/" text="Components" %}
-        {% gds_header_nav_item href="/" text="User list" %}
+        {% gds_header_nav_item href="/" text="Home" %}
+        {% gds_header_nav_item href="/data-products" text="Data products" %}
```

Comment:
@michaeljcollinsuk , just tried ,  then my memory has been recalled ,  yes I tried it, but somehow cannot be recognised in the following line 

```
{% gds_header_nav_item href="/data-products" text="Data products" %}
```
my guess it is due to nested `{%  {% %}  %}`,  how about  I keep this way, you may find it out more in your PR

Reply:
Good point, leave as is for now and I will look into this as part of my frontend work

Reply:
You may be able to do like this:
```
{% url 'data-products' as data_products_url %}
{% gds_header_nav_item href=data_products_url text="Data products" %}
```

Reply:
Think the best way is to add them as context vars in the view, but we can leave it for now and come back to it later

Reply:
context vars is even worse 😭 ,   alright I will give try, but I feel this way won't work

Reply:
Why are context vars worse? I mean by adding them into the view `get_context_data` method. Surely that isn't be worse than hardcoding the url in the template?

Reply:
But also I am not too worried about leaving them there for now, as I know these templates will change in future anyway

Reply:
@michaeljcollinsuk,  I feel you are 🔥  at me --> `why are context vars worse!!!! tell me` 😃 
  tried, it won't work as I expected,   the problem is that even you defined a temporary var ,  but in template , if you want the engine to convert it we have to use `{% %}`,   so if you just add `data_products_url` in the above, the engine will treat it as plain text

Reply:
The reason for not wanting to provide this way to make  it work , is that we somehow have to do extra code (which is supposed to be supported by django template engine by default) for those url,  also I am concerning we will end up to repeat to define those context vars just because of using this component 😊

Reply:
😆😆😆 not at all, just curious. Hmm what if you use `{{ data_products_url }}` or does that error also? Sounds like it is getting too much hassle!

Reply:
```
Could not parse the remainder: '{{' from '{{'
```
-_-!

Reply:
if this way doesn't work, passing via context var won't work too

Reply:
Only way I managed to get it work:
```
{% block header %}
    {% url "data-products" as data_products_url %}
    {% gds_header homepageUrl="/" serviceName=durl serviceUrl="/" containerClasses="govuk-width-container" %}
        {% gds_header_nav_item href="/" text="Home" %}
        {% gds_header_nav_item href=data_products_url text="Data products" %}
    {% endgds_header %}
```
But as you say, maybe not ideal. In future could add a context processor that passes some urls and any other info needed within the base.html template. But maybe you think that is overkill 😆

Reply:
hmm are you sure the above one work?, why didn't work forme?

Reply:
Might have been that you did not define `{% url "data-products" as data_products_url %}` within the `header` block?

Reply:
Or using a context processor:
```
# in new file e.g interfaces/web/utils.py
from django.urls import reverse


def header_context(request):
    return {
        "data_products_url": reverse("data-products")
    }


# in common settings file, update the TEMPLATES context_processors dict:

            "context_processors": [
                "django.template.context_processors.debug",
                "django.template.context_processors.request",
                "django.contrib.auth.context_processors.auth",
                "django.contrib.messages.context_processors.messages",
                "controlpanel.interfaces.web.utils.header_context"
            ],
            
# then in the base.html template (or any other template) you can use data_products_url without brackets etc
{% gds_header_nav_item href=data_products_url text="Data products" %}
```

Reply:
alright,  found the difference,  the temporary var has been defined with the `{% block %}`  🙃 

for url part, if we can do the above way,  then not too bad,  another thing is that that repo is quite new,   a little bit concern how this https://github.com/uktrade/govuk-frontend-django will be maintain,  anyway, hopefully it will be very useful for form side